autoload/test/r/testthat.vim

```diff
@@ -0,0 +1,49 @@
+if !exists('g:test#r#testthat#file_pattern')
+  let g:test#r#testthat#file_pattern = '\v(test-[^/]+|[^/]+)\.R$'
```

Comment:
Is this regex case sensitive? If so, I'd like to support lowercase `.r` files as well because case insensitive filesystem users often play fast and loose with casing in filenames. (I myself am guilty of this ✋)

```suggestion
  let g:test#r#testthat#file_pattern = '\v(test-[^/]+|[^/]+)\.[Rr]$'
```